build/Tests/3rdparty/libDispatch/libDispatch.3rdPartyUT.vcxproj

```diff
@@ -221,34 +221,34 @@
-    <ClangCompile Include="..\..\..\..\tests\unittests\libDispatch\dispatch_drift.m" />
-    <ClangCompile Include="..\..\..\..\tests\unittests\libDispatch\dispatch_group.m" />
-    <ClangCompile Include="..\..\..\..\tests\unittests\libDispatch\dispatch_pingpong.m" />
-    <ClangCompile Include="..\..\..\..\tests\unittests\libDispatch\dispatch_priority.m" />
-    <ClangCompile Include="..\..\..\..\tests\unittests\libDispatch\dispatch_sema.m" />
-    <ClangCompile Include="..\..\..\..\tests\unittests\libDispatch\dispatch_starfish.m" />
-    <ClangCompile Include="..\..\..\..\tests\unittests\libDispatch\dispatch_test.m" />
-    <ClangCompile Include="..\..\..\..\tests\unittests\libDispatch\dispatch_timer_bit31.m" />
-    <ClangCompile Include="..\..\..\..\tests\unittests\libDispatch\dispatch_timer_bit63.m" />
-    <ClangCompile Include="..\..\..\..\tests\unittests\libDispatch\dispatch_timer_set_time.m" />
-    <ClangCompile Include="..\..\..\..\tests\unittests\libDispatch\dispatch_autorelease.m" />
+    <ClangCompile Include="$(StarboardBasePath)\tests\3rdparty\libDispatch\dispatch.m" />
+    <ClangCompile Include="$(StarboardBasePath)\tests\3rdparty\libDispatch\dispatch_after.m" />
+    <ClangCompile Include="$(StarboardBasePath)\tests\3rdparty\libDispatch\dispatch_api.m" />
+    <ClangCompile Include="$(StarboardBasePath)\tests\3rdparty\libDispatch\dispatch_apply.m" />
+    <ClangCompile Include="$(StarboardBasePath)\tests\3rdparty\libDispatch\dispatch_c99.m" />
+    <ClangCompile Include="$(StarboardBasePath)\tests\3rdparty\libDispatch\dispatch_cascade.m" />
+    <ClangCompile Include="$(StarboardBasePath)\tests\3rdparty\libDispatch\dispatch_debug.m" />
+    <ClangCompile Include="$(StarboardBasePath)\tests\3rdparty\libDispatch\dispatch_drift.m" />
+    <ClangCompile Include="$(StarboardBasePath)\tests\3rdparty\libDispatch\dispatch_group.m" />
+    <ClangCompile Include="$(StarboardBasePath)\tests\3rdparty\libDispatch\dispatch_pingpong.m" />
+    <ClangCompile Include="$(StarboardBasePath)\tests\3rdparty\libDispatch\dispatch_priority.m" />
+    <ClangCompile Include="$(StarboardBasePath)\tests\3rdparty\libDispatch\dispatch_sema.m" />
+    <ClangCompile Include="$(StarboardBasePath)\tests\3rdparty\libDispatch\dispatch_starfish.m" />
+    <ClangCompile Include="$(StarboardBasePath)\tests\3rdparty\libDispatch\dispatch_test.m" />
+    <ClangCompile Include="$(StarboardBasePath)\tests\3rdparty\libDispatch\dispatch_timer_bit31.m" />
+    <ClangCompile Include="$(StarboardBasePath)\tests\3rdparty\libDispatch\dispatch_timer_bit63.m" />
+    <ClangCompile Include="$(StarboardBasePath)\tests\3rdparty\libDispatch\dispatch_timer_set_time.m" />
+    <ClangCompile Include="$(StarboardBasePath)\tests\3rdparty\libDispatch\dispatch_autorelease.m" />
   </ItemGroup>
   <ItemGroup>
-    <ClInclude Include="..\..\..\..\tests\unittests\libDispatch\dispatch_test.h" />
+    <ClInclude Include="$(StarboardBasePath)\tests\3rdparty\libDispatch\dispatch_test.h" />
```

Comment:
Synced offline and decided to not include this project in build.sln

---
In reply to: [74129942](https://github.com/Microsoft/WinObjC/pull/853#discussion_r74129942) [](ancestors = 74129942,74129569)